docs/content/how-to/sdn/other-sdn-providers.md

```diff
@@ -0,0 +1,402 @@
+This document explains how to create a HostedCluster that runs an SDN provider different from OVNKubernetes. The document assumes that you already have the required infrastructure in place to create HostedClusters.
+
+> **IMPORTANT**: The work described here is **not supported**. SDN providers **must** certify their software on HyperShift before it becomes a supported solution. The steps described here are just a technical reference for people who wants to try different SDN providers in HyperShift.
+
+Versions used while writing this doc:
+
+- Management cluster running OpenShift `v4.14.6` and HyperShift Operator version `e87182ca75da37c74b371aa0f17aeaa41437561a`.
+- HostedCluster release set to OpenShift `v4.14.8`.
+
+## Calico
+### Deployment
+
+In this scenario we are using the Calico version v3.27.0 which is the last one at the time of this writing. The steps followed rely on the [docs](https://docs.tigera.io/calico/latest/getting-started/kubernetes/openshift/installation#generate-the-install-manifests) by Tigera to deploy Calico on OpenShift.
+
+1. Create a `HostedCluster` and set its `HostedCluster.spec.networking.networkType` to `Calico`.
+
+2. Wait for the HostedCluster's API to be ready. Once it's ready, get the admin kubeconfig.
+
+3. Wait for compute nodes to join the cluster. Keep in mind since the SDN is not deployed yet, they will remain in `NotReady` state.
```

Comment:
I don't think it's necessary to wait for nodes to join. You should be able to apply the manifests as long as you have a working API server.

Reply:
fixed